prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var firstElement = languages[0];
+console.log('the first element of the array is', firstElement);
+const thirdElement = languages[languages.length - 4];
+console.log('the 3rd element of the array is', thirdElement);
+var length = 6;
+console.log('The length of the array is', length);
```

Comment:
Hi Mohammed, you can get the length of the array from its `.length` property.

```js
var length = languages.length;
```

Reply:
<img width="1682" alt="Screen Shot 2022-03-04 at 2 47 39 PM" src="https://user-images.githubusercontent.com/99572164/156852470-197aa0d3-6eda-4d06-9cdd-817c3bf08865.png">